src/Management/EventStoreManagement.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Depot\EventStore\Management;
+
+use Depot\EventStore\EventVisitor;
+
+interface EventStoreManagement
+{
+    public function visitEvents(Criteria $criteria, EventVisitor $eventVisitor);
```

Comment:
Let's call this `visitCommittedEvents`. (and the second param will become `CommittedEventVisitor $committedEventVisitor`)